test/test_b4nsd.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Could you consider s/b4ns/b4nn/g ?

Because we abbreviate slirp4netns as "s4nn": https://github.com/rootless-containers/rootlesskit/blob/877719759621b2e35b5bd8317accc667eff9126b/cmd/rootlesskit/main.go#L322